source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextBlockRenderer.mm

```diff
@@ -195,6 +196,40 @@ + (NSNumber *)getTextStrokeWidthForWeight:(TextWeight)weight
             return @-2;
     }
 }
+// given string with short form of date (MM/dd/yyyy), convert it to
+// NSDateFormatterMediumStyle or NSDateFormatterLongStyle in local language
++ (void) getLocalizedDate:(std::shared_ptr<TextBlock> const &)txtBlck stringWithDate:(std::string &)dateParsedString
```

Comment:
>dateParsedString [](start = 100, length = 16)

It's weird that this function takes `dateParsedString` as an in-out parameter and appends to it. I think it would make more sense just to return the formatted string as a return value.

Reply:
sure, that makes more sense.

---
In reply to: [171674341](https://github.com/Microsoft/AdaptiveCards/pull/1194#discussion_r171674341) [](ancestors = 171674341)